hw/ip/csrng/data/csrng.hjson

```diff
@@ -571,5 +571,43 @@
         },
       ]
     },
+    { name: "SEL_TRACKING_SM",
+      desc: "Select debug tracking state machine register",
+      swaccess: "wo",
+      hwaccess: "hro",
+      fields: [
+        {
+            bits: "1:0",
+            name: "SEL_TRACKING_SM",
+            desc: '''
+                  These encoded bits will select one of four groups of
+                  state machine tracking registers. Each tracking debug field is 8 bits
+                  wides, so four debug fields will be observed at one time.
+                  Each tracking field will will follow a per instance csrng application command
```

Comment:
duplicate "will"

Reply:
ah thanks for finding the nits, my editor does not check for that stuff currently.